Mobile/ShopLens/Droid/Activities/MainActivity.cs

```diff
@@ -22,11 +26,17 @@ public enum ActivityIds
 
 namespace ShopLens.Droid
 {
-    [Activity(Label = "ShopLens", MainLauncher = true, Icon = "@mipmap/icon")]
-    public class MainActivity : Activity, IRecognitionListener
+    [Activity(Label = "ShopLens", MainLauncher = true, Icon = "@mipmap/icon", Theme ="@style/ShopLensTheme")]
```

Comment:
Do you really use this icon? I think it's better to delete it.

Reply:
It is used actually, you can find it in mipmap folders and it is how our app's icon looks on phone. We should create a new icon though.